core/trino-main/src/test/java/io/trino/operator/scalar/TestIpAddressFunctions.java

```diff
@@ -75,7 +75,7 @@ public void testIpAddressContains()
         assertThat(assertions.function("contains", "'0.0.0.0/4'", "IPADDRESS '0.0.0.0'"))
                 .isEqualTo(true);
         assertThat(assertions.function("contains", "'16.0.0.0/4'", "IPADDRESS '16.0.0.0'"))
-                .isEqualTo(true);
+                        .isEqualTo(true);
```

Comment:
Revert.

Reply:
hmm weird, my editor must have done that.